pysd/py_backend/xmile/xmile2py.py

```diff
@@ -43,7 +43,7 @@ def get_xpath_text(element, path, ns={'ns': NS}, default=''):
 
     model_elements = []
     smile_parser = SMILEParser(namespace)
-
+    
     # add aux and flow elements
     flaux_xpath = '//ns:model/ns:variables/ns:aux|//ns:model/ns:variables/ns:flow'
     for element in root.xpath(flaux_xpath, namespaces={'ns': NS}):
```

Comment:
`element` here may want to be `node` as you have in line 72

Reply:
Oh yes, I think that this file require some more changes and refactorings :)

Reply:
Sounds good. Let me know when you're ready for me to merge.